apps/Minecraft Java Server/install

```diff
@@ -66,6 +66,21 @@ case "$server_type" in
     setsid x-www-browser "https://files.minecraftforge.net/net/minecraftforge/forge/" > /dev/null 2>&1 &
     server_jar="forge-*-installer.jar"
     ;;
+    "Paper")
+    # open browser to download the server version of user choice
+    setsid x-www-browser "https://papermc.io/downloads/" > /dev/null 2>&1 &
```

Comment:
remove the `/` after downloads or else you get a 404

Reply:
do you mind fixing it I am busy right now

Reply:
I don't have permissions on this repo to fix myself

Reply:
@theofficialgman I invited you to be a collaborator. I'm still busy